crates/api_common/src/post.rs

```diff
@@ -164,6 +165,7 @@ pub struct EditPost {
   /// Instead of fetching a thumbnail, use a custom one.
   #[cfg_attr(feature = "full", ts(optional))]
   pub custom_thumbnail: Option<String>,
+  pub tags: Option<Vec<TagId>>,
```

Comment:
Dont see the need for Option here, you can simply use an empty vec if there are no tags.

Reply:
Use `#[serde(default)]` then clients dont need to set this field.